src/components/PostEditor/PostEditor.test.js

```diff
@@ -27,6 +27,7 @@ describe('PostEditor', () => {
   it('renders announcement option with admin in props', () => {
     const props = {
       ...baseProps
+      // KMT: where is admin?
```

Comment:
hmm, gooood question

Reply:
Yeah, there's no `admin` option in the `PostEditor.js` file. Is it an external influence?

Reply:
what is meant by admin here is actually canModerate. you could get the test setup correctly to show the editor as a group moderator and then it should show the announcement option.